packages/embark/src/cmd/cmd_controller.js

```diff
@@ -285,6 +286,7 @@ class EmbarkController {
         engine.registerModulePackage('embark-deploy-tracker', { plugins: engine.plugins });
 
         engine.registerModuleGroup("blockchain");
+        engine.registerModulePackage('embark-suggestions');
```

Comment:
Don't think we need suggestions in `build`

Reply:
agree